ext_emconf.php

```diff
@@ -8,14 +8,13 @@
     'author_email' => 'schlosser@itplusx.de',
     'author_company' => 'ITplusX GmbH',
     'state' => 'stable',
-    'clearCacheOnLoad' => 0,
-    'version' => '3.0.0-dev',
+    'version' => '4.0.0-dev',
```

Comment:
I'd rather stay at 3.0.0-dev since v3.0.0 was never released yet.

Reply:
@schloram, makes sense. Reverted that change!